frontend/packages/dev-console/src/components/EmptyState.tsx

```diff
@@ -23,14 +25,16 @@ const navigateTo = (e: Event, url: string) => {
   e.preventDefault();
 };
 
-const ODCEmptyState: React.FC<Props> = ({ title, activeNamespace }) => {
+const ODCEmptyState: React.FC<Props> = ({
+  title,
+  activeNamespace,
+  hintBlock = 'Select a way to create an application, component or service from one of the options.',
+}) => {
   return (
     <React.Fragment>
       <div className="odc-empty-state__title">
         <PageHeading title={title} />
-        <p className="co-catalog-page__description">
-          Select a way to create an application, component or service from one of the options.
-        </p>
+        <div className="co-catalog-page__description odc-empty-state__hint-block">{hintBlock}</div>
```

Comment:
If you want to allow this to be option, I suggest a more simple:
```suggestion
        {hintBlock && (
          <div className="co-catalog-page__description odc-empty-state__hint-block">
            {hintBlock}
          </div>
        )}
```

That way they can either provide `null` or `''` and it will not render the padding with no content.